src/site/content/zh/metrics/ttfb/index.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: post
+title: Time to First Byte 第一字节时间 (TTFB)
+authors:
+  - jlwagner
+date: 2021-10-26
+updated: 2022-03-30
+description: |
+  本篇文章介绍了第一字节时间 (TTFB) 指标并说明了该指标的测量方式
+tags:
+  - performance
+  - metrics
+  - web-vitals
+---
+
+{% Aside %}
+第一字节时间 (TTFB) 是在实验室和现场测量连接建立时间和 Web 服务器响应能力的一个基础指标。它有助于识别 Web 服务器何时对请求的响应速度太慢。对 HTML 文档的请求，该指标先于其他所有的加载性能指标。
+{% endAside %}
+
+## TTFB 是什么?
+
+TTFB 是一个衡量对资源的请求和响应的第一个字节开始和到达之间时间的指标。
+
+<figure>
+  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/ccT8ltSPrTri3tz7AA3h.png", alt="A diagram of network request timings. The phases from left to right are Redirect (which overlaps with Prompt for Unload), Cache, DNS, TCP, Request, Response, Processing, and Load. The associated timings are redirectStart and redirectEnd (which overlap with the Prompt for Unload's unloadEventStart and unloadEventEnd), fetchStart, domainLookupStart, domainLookupEnd, connectStart, secureConnectionStart, connectEnd, requestStart, responseStart, responseEnd, domInteractive, domContentLoadedEventStart, domContentLoadedEventEnd, domComplete, loadEventStart, and loadEventEnd.", width="800", height="337" %}
+  <figcaption>
+    网络请求阶段及其相关时间损耗的图示. TTFB 测量<code>startTime</code>和<code>responseStart</code>之间的时间损耗。
+  </figcaption>
+</figure>
+
+TTFB 是下列请求节点的时间损耗汇总：
+
+- Redirect time 重定向时延
+- Service worker 启动时延（如果适用）
+- DNS 查询时延
+- 建立连接和 TLS 所消耗时延
+- 请求，直到响应的第一个字节到达为止的时延
+
+减少连接建立和后端服务的时延将有助于降低 TTFB
+
+### 什么是好的 TTFB 分数?
+
+由于网络与后端服务的差异很大，因此不能对什么是“好的” TTFB 分数设定一个任意的数字。由于 TTFB 先于[以用户为中心的指标](/user-centric-performance-metrics/)，如[首次内容显示（FCP）](/fcp/)和[最大内容显示（LCP）](/lcp/)，因此建议你的服务器对请求的响应速度要足够快，以使 **P75** 的用户体验在[良好的 FCP 分数](/fcp/#what-is-a-good-fcp-score).
```

Comment:
```suggestion
由于 TTFB 先于[以用户为中心的指标](/user-centric-performance-metrics/)，如[首次内容显示（FCP）](/fcp/)和[最大内容显示（LCP）](/lcp/)，因此建议你的服务器对请求的响应速度要足够快，以使 **P75** 的用户体验在[良好的 FCP 分数](/fcp/#what-is-a-good-fcp-score).
```

Reply:
I think the original post updated since the last translation. I don't see this line in the english post.